packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -22,27 +22,141 @@
-        >
+        <div v-if="fetching" class="flex justify-center">
           <HoppSmartSpinner />
         </div>
 
         <div v-else-if="error">{{ t('users.load_list_error') }}</div>
 
-        <UsersTable
-          v-else-if="usersList.length >= 1"
-          :usersList="usersList"
-          :fetching="fetching"
-          :error="error"
-          @goToUserDetails="goToUserDetails"
-          @makeUserAdmin="makeUserAdmin"
-          @makeAdminToUser="makeAdminToUser"
-          @deleteUser="deleteUser"
-        />
+        <div v-else-if="usersList.length">
+          <HoppSmartTable :list="usersList">
+            <template #head>
+              <tr
+                class="text-secondary border-b border-dividerDark text-sm text-left bg-primaryLight"
+              >
+                <th class="px-6 py-2">{{ t('users.id') }}</th>
+                <th class="px-6 py-2">{{ t('users.name') }}</th>
+                <th class="px-6 py-2">{{ t('users.email') }}</th>
+                <th class="px-6 py-2">{{ t('users.date') }}</th>
+                <!-- Empty header for Action Button -->
+                <th class="px-6 py-2"></th>
+              </tr>
+            </template>
+
+            <template #body="{ list }">
+              <tr
+                v-for="user in list"
+                :key="user.uid"
+                class="text-secondaryDark hover:bg-divider hover:cursor-pointer rounded-xl"
+                @click="goToUserDetails(user.uid)"
+              >
+                <td class="py-2 px-3 max-w-30">
+                  <div class="flex">
+                    <span class="truncate">
+                      {{ user.uid }}
+                    </span>
+                  </div>
+                </td>
+
+                <td class="py-2 px-3">
+                  <div v-if="user.displayName" class="flex items-center">
+                    <span>
+                      {{ user.displayName }}
+                    </span>
+                    <span
+                      v-if="user.isAdmin"
+                      class="text-xs font-medium ml-2 px-3 py-0.5 rounded-full bg-green-900 text-green-300"
+                    >
+                      {{ t('users.admin') }}
+                    </span>
+                  </div>
+                  <div v-else class="flex items-center space-x-3">
+                    <span> {{ t('users.unnamed') }} </span>
+                    <span
+                      v-if="user.isAdmin"
+                      class="text-xs font-medium px-3 py-0.5 rounded-full bg-green-900 text-green-300"
+                    >
+                      {{ t('users.admin') }}
+                    </span>
+                  </div>
+                </td>
+
+                <td class="py-2 px-3">
+                  {{ user.email }}
+                </td>
 
-        <div v-else class="flex justify-center">{{ t('users.no_users') }}</div>
+                <td class="py-2 px-3">
+                  <div class="flex flex-col">
```

Comment:
```suggestion
                  <div>
```

Isn't a single column the default layout? Could you verify it the behavior remains the same without the classes applied?